hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusCreateTopicHandler.java

```diff
@@ -174,6 +173,63 @@ public void handle(@NonNull final HandleContext handleContext) {
         }
     }
 
+    private void validateSemantics(
```

Comment:
It is clear if we don't pass `Builder` into this method and just validations happen here. After this we construct the builder

Reply:
Done